api/v1/stocks.go

```diff
@@ -29,24 +30,46 @@ func (api *APIHandler) getAllStocks(w http.ResponseWriter, r *http.Request) {
 
 func (api *APIHandler) getStockDetails(w http.ResponseWriter, r *http.Request) {
 	symbol := chi.URLParam(r, "symbol")
+	time := chi.URLParam(r, "time")
 
 	yamlData, err := details.StocksYamlBytes()
```

Comment:
If we load stock details using `fantasymarket/game/details.LoadStockDetails`, we'd get the results as a map and thus can prevent reimplementing it using the `getStockHelper`.

Even better, we could use api.Game.StockDetails which uses this function under the hood. This way, we wouldn't need to generate the array for every request.

Reply:
🤯 Thank you so much for the constructive feedback! I wasn't aware of api.Game.StockDetails, I will look into that! :0